doc/source/io.rst

```diff
@@ -1994,6 +1994,18 @@ into a flat table.
 .. ipython:: python
 
    from pandas.io.json import json_normalize
+   data = [
+             {'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
+             {'name': {'given': 'Mose', 'family': 'Regner'}},
+             {'id': 2, 'name': 'Faye Raker'}
+          ]
+   json_normalize(data)
+
+.. _io.jsonl:
```

Comment:
Gotcha, that looks good (based on screenshot below). I think you still want to remove this line though, otherwise it's creating a reference anchor to this point.